targets/ath79-generic

```diff
@@ -10,6 +10,9 @@ local ATH10K_PACKAGES_QCA9888 = {
 device('devolo-wifi-pro-1200i', 'devolo_dvl1200i', {
 	factory = false,
 })
+device('devolo-wifi-pro-1750i', 'devolo_dvl1750i', {
+	factory = false,
```

Comment:
ath10k package reference is missing here - See https://github.com/freifunk-gluon/gluon/commit/e30b0092bb00ea51d3d08838b6ab0c401095fc3a